src/wallet/coin_selection.rs

```diff
@@ -597,6 +638,7 @@ impl BranchAndBoundCoinSelection {
         CoinSelectionResult {
             selected,
             fee_amount,
+            excess: Excess::Change(0, 0),
```

Comment:
I suppose these are just temporary values, right?

Reply:
Yes, you're right. I'm trying to find a solution with a good balance between the parameter increment and the generality to reach each of the possible outputs of `BranchAndBound`.